include/tester/core/Application.hpp

```diff
@@ -60,15 +54,21 @@ class Application
 
     bsoncxx::document::value getHistoryHeaders()
```

Comment:
rename this method name to `getTestLogsHistory()` too

Reply:
Done